src/team-view/team-view.html

```diff
@@ -38,19 +44,63 @@
 
         <paper-button raised
                       on-tap="_next">Play Again</paper-button>
+        <div id="fb-share-btn">
+          <social-media-icons icon="facebook" raised on-tap="shareClick"></social-media-icons>
+        </div>
+        <!-- </paper-button> -->
     </template>
     <script>
         Polymer({
             is: 'team-view',
             behaviors: [ViewBehavior],
             properties: {
                 mapData: Object,
-                _spiritNames: String
+                _spiritNames: String,
+                shareText: {
+                  type: String,
+                  computed: 'generateText(_spiritNames)'
+                }
             },
             observers: [
                 '_appStateChanged(appState)'
             ],
 
+            ready: function() {
+              window.fbAsyncInit = function() {
+                FB.init({
+                  appId      : '1923680841195910',
```

Comment:
If we go this route, we need to figure out how to share this ID with my Facebook development account so that I can maintain the project after you're gone.